### PR TITLE
Release/r158

### PR DESCRIPTION
**Pull request by benjben**

### CODE REVIEW

**Review by colmsnowplow — APPROVED** [review]

File: schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        "errors": {
+            "type": "array",
+            "items": {
+                "type": "object"
+            },
+            "description": "A list of errors encountered, and supporting information about the error. Each entry should always have at least a message field and a source field."
+        },
+        "schema": {
+					"type": ["string", "null"],
+					"description": "The schema for the object which caused the failure, if it was self-descrbing."
```
Comment (istreeter) [inline schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0:24]: Weird formatting??

**Review by istreeter — APPROVED** [review]

File: schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0 (hunk 1 above)
Comment (jbeemster) [reply schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0:24]: @benjben did you want to fix that up?

Comment (benjben) [reply schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0:24]: Yes !

Comment (benjben) [reply schemas/com.snowplowanalytics.snowplow/failure/jsonschema/1-0-0]: Done!

**Review by jbeemster — APPROVED** [review]

**Pull request merged**